jme3-core/src/main/java/com/jme3/anim/tween/action/BlendableAction.java

```diff
@@ -7,6 +38,20 @@
 import com.jme3.util.clone.Cloner;
 import java.util.Collection;
 
+/**
+ * An implementation of the Action interface, used to manage the interpolation code
```

Comment:
Quibble: Action is not an interface.  This is a "specialization" of the Action "base class".

Reply:
Great, I am going to remove all the architectural keywords, since the original author is not here, I will try to stick more to describing the behavior of the system.